gateway/sds_gateway/visualizations/cog_pipelines.py

```diff
@@ -517,23 +517,33 @@ def visualization_error_handler(error, task_run=None):
                     ProcessingStatus,
                 )
 
-                # Find any PostProcessedData records for this capture that are still
-                # pending
+                # Find any PostProcessedData records for this capture that are pending
+                # or processing
                 failed_records = PostProcessedData.objects.filter(
                     capture__uuid=capture_uuid,
-                    processing_status=ProcessingStatus.Pending.value,
+                    processing_status__in=[
+                        ProcessingStatus.Pending.value,
+                        ProcessingStatus.Processing.value,
+                    ],
                 )
 
+                if not failed_records:
+                    logger.warning(
+                        f"No PostProcessedData records found for capture {capture_uuid}"
+                        f" that are pending or processing"
+                    )
+                    return
+
                 for record in failed_records:
                     error_message = f"COG task '{task_run.task.name}' failed: {error}"
                     record.mark_processing_failed(error_message)
-                    logger.info(
+                    logger.warning(
                         f"Marked PostProcessedData record {record.uuid} as failed "
                         f"due to COG task failure"
                     )
 
             else:
-                logger.warning(
+                logger.error(
                     "Could not extract capture_uuid from pipeline run arguments"
                 )
```

Comment:
if this happens, is that an indication something is wrong with the code or pipeline that needs our attention? If not, I'd keep it as a warning.